src/jit-rl-cse/MLCSE.cs

```diff
@@ -675,6 +682,43 @@ static void PolicyGradient(IEnumerable<Method> methods)
                                     string dasmRun = SPMI.Run(method.spmiIndex, dasmOptions);
                                     sw.WriteLine($" ---> saved baseline dasm to {baseDasmFile}");
                                 }
+
+                                string greedySequence = "greedy";
+                                string greedyDumpFile = Path.Combine(dumpDir, $"dump-{method.spmiIndex}-{greedySequence}-{r}.d");
+
+                                if (File.Exists(greedyDumpFile))
```

Comment:
nit: `File.Delete(file)` doesn't throw when file doesn't exist, so you don't need `Exists` check in such cases 🙂

Reply:
Thanks. I'll be messing with this code more in the future, I want it to be able to spot when it has problems and automatically collect the right dumps, instead of me having to tell it which ones I want.